### ISSUE TITLE
1.1 Release for the AWS Quick Start

### ISSUE THREAD

**Issue by s0enke** [superwerker/superwerker#186]
Create a PR for the AWS Quickstart repo once the 1.1 milestone is finished

**Comment by s0enke** [comment]
Created a commit for a new QuickStart release: https://github.com/aws-quickstart/quickstart-superwerker/commit/fd13a2991094af5b59ea50dfcd606b506782e7fb
Waits for #200 to be reviewed and merged.

**Comment by s0enke** [comment]
Waiting for the successful pipeline run for a screenshot.

**Comment by s0enke** [comment]
PR: https://github.com/aws-quickstart/quickstart-superwerker/pull/23

**Comment by s0enke** [comment]
PR has been merged into develop -> https://github.com/aws-quickstart/quickstart-superwerker/tree/develop
Now waiting for the Quick Start team to merge to `main`.
